app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1315,6 +1315,87 @@ function (&$field) {
         return $this->viewAction($objectId);
     }
 
+
+    /**
+     * Send example email to many users
+     *
+     *
+     * @return \Symfony\Component\HttpFoundation\JsonResponse|\Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function exampleMultipleAction()
+    {
+        if ($this->request->getMethod() == 'POST') {
+
+            $emails = $this->request->request->get('emails');
+
+            $objectId = 1;
+
+            /** @var \Mautic\EmailBundle\Model\EmailModel $model */
+            $model  = $this->getModel('email');
+            $entity = $model->getEntity($objectId);
+
+            //not found or not allowed
+            if ($entity === null
+                || (!$this->factory->getSecurity()->hasEntityAccess(
```

Comment:
Please use 
`$this->security`
instead of 
`$this->factory->getSecurity()`
Because the Mautic Factory is deprecated. We want to get rid of it to make Mautic easily testable.